### PR TITLE
tests - watchOrderBookForSymbols, watchTradesForSymbols, watchOHLCVForSymbols

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

**Comment by ttodua** [comment]
@carlosmiei this one passes too

**Comment by ttodua** [comment]
@carlosmiei if you'd find some time for this PRs would be good, so I could add new methods for them across exchanges more reliably

**Comment by ttodua** [comment]
@carlosmiei can you look into this?

**Comment by carlosmiei** [comment]
@ttodua the build has been to unstable, not sure we should add more tests until we stabilize it

**Comment by ttodua** [comment]
, watch .FOrSymbols are so much important methods for most exchanges and almost everything rely on them, we should not skip/sacrifice them in expense of builds. also, they are mostly implemented for toptier exchanges, not for crappies ones to expect frequent failures.
but oky, as you decide.

**Comment by ttodua** [comment]
@carlosmiei in this PR, i've added a feature to skip language-specific test (i.e not `skipPhpAsync` or `skipCSharp`, but `watchOrderBook.cs` which means to affect only Csharp tests for watchOrderBook. this would be important to handle travis failures with more flexibility

**Comment by ttodua** [comment]
merged another PR into this, which gives us more control over tests, to avoid accidental skips of important parts

**Comment by ttodua** [comment]
@carlosmiei this too

File: run-tests.js [hunk 1]
```diff
@@ -266,54 +266,55 @@ const testExchange = async (exchange) => {
...
+    if (!wsFlag) {
+        allTestsWithoutTs.push (
+            { language: 'PHP',        key: '--php',          exec: ['php', '-f', 'php/test/test_sync.php',         ...args] } 
+        );
+        allTestsWithoutTs.push (
+            { language: 'Python 3',   key: '--python',       exec: ['python3',   'python/ccxt/test/test_sync.py',  ...args] }
+        );
+    }
 
-        const allTests = allTestsWithoutTs.concat([
```
Comment (ttodua) [inline run-tests.js:291]: i have not removed the codes from here down below, just git shows them into differences

Comment (ttodua) [reply run-tests.js:291]: so, there are no changed from here down below, other than one indent was removed (which was extra)

**Comment by carlosmiei** [comment]
`Would like to see a green build first

**Comment by ttodua** [comment]
> `Would like to see a green build first

I think it might not pass, because binance's watchOB has been consistently failing in c# , so after this PR gets merged I can immediately add skip ob specifically for `c#`

**Comment by ttodua** [comment]
i mean, inside this PR , skips.json might not be filled as it needs, however, this PR would give us opportunity to add quick skip-PRs easily afterwards

**Comment by carlosmiei** [comment]
> because binance's watchOB has been consistently failing in c#
@ttodua in that case, maybe we should tackle this first, merge that PR, and rebase this one to see if it passes with the fix

**Comment by ttodua** [comment]
I don't think so, that problem is a separate problem, which blocks the development (and not vice-versa), so that problem should be "muted" until that problem is fixed. stopping development before that problem is fixed at first, is not correct thing imho.

**Comment by carlosmiei** [comment]
@ttodua yes the problem is a different one, but one that directly impacts users whereas we might argue that adding tests does not immediately affect the users (the beneficial effect is more long-term)

**Comment by ttodua** [comment]
I might be a little confused, how unrelated issues prevent tests updates for merging (while this is what exposes issues and "help users" from errors) even though builds might be failing for unrelated things (and even if related issues, we can skip temporarily and work on the fix parallely), but sure, as you consider it better ✌️

**Comment by ttodua** [comment]
pushed 2 PRs which were failing here (for bitget and deribit)

**Comment by ttodua** [comment]
pushed 3rd PR too , to fix the travis fails here.
the remaining issues are also purely unrelated and all they come from c#

**Comment by ttodua** [comment]
anyway, I've merged Poloniex Fetchtickers fix PR here too, to make build pass here too

**Comment by ttodua** [comment]
if you haven't started a review, i'll turn this into draft, because other PR (multi-build travis) also modifies run-tests and there might be conflicts and might be annyong for your re-review, so let's try to finish that PR at first.

**Comment by ttodua** [comment]
making last commit for run-tests in 2 mins.

**Pull request merged**